src/vscode-bicep/src/language/client.ts

```diff
@@ -62,12 +66,13 @@
     clientOptions
```

Comment:
>unable [](start = 106, length = 6)

I actually like that it's config based. This way we don't need to ask users to install some debug build, which may not even repro a particular issue.